contracts/staking/SimpleStaking.sol

```diff
@@ -120,26 +120,28 @@ contract SimpleStaking is DSMath, Pausable, AbstractGoodStaking, StakingToken{
         if (tokenActual < tokenWithdraw) {
             tokenWithdraw = tokenActual;
         }
-        uint256 gdInterest =  InterestDistribution.withdrawStakeAndInterest(interestData, msg.sender, _amount);
+        //uint256 gdInterest =  InterestDistribution.withdrawStakeAndInterest(interestData, msg.sender, _amount);
         //Since we use generic ERC20 function we can just use its interface
-        ERC20 goodDollar = ERC20(address(avatar.nativeToken()));
-        fm.mintReward(msg.sender); // send rewards to user
+        //ERC20 goodDollar = ERC20(address(avatar.nativeToken()));
+        fm.mintReward(address(iToken), msg.sender); // send rewards to user
         _burn(msg.sender, _amount); // burn their staking tokens
         _decreaseProductivity(msg.sender, _amount);
-        require(goodDollar.transfer(msg.sender, gdInterest), "withdraw interest transfer failed");
+        //require(goodDollar.transfer(msg.sender, gdInterest), "withdraw interest transfer failed");
         require(token.transfer(msg.sender, tokenWithdraw), "withdraw transfer failed");
```

Comment:
need to convert iToken to token in order to withdraw.
add an issue to allow user to withdraw either token or iToken

Reply:
we redeem it